docker-compose.yml

```
@@ -0,0 +1,9 @@
+version: "3.9"
+services:
+  app:
+    build: .
```

This is implicitly using the last stage in your Dockerfile, which is unreliable. The docker-compose equivalent to the CLI's `--target development` build option looks like this:
```suggestion
    build: 
      context: .
      target: development
```